charts/redpanda/console.tpl.go

```diff
@@ -236,6 +231,11 @@ func ConsoleConfig(dot *helmette.Dot) map[string]any {
 				"tls": values.Listeners.Admin.ConsoleTLS(&values.TLS),
 			},
 		},
+		"schemaRegistry": map[string]any{
```

Comment:
This is going to be a bit frustrating but this change will have to be submitting in a follow up PR. The charts are versioned separately (and the release infra is a bit odd) so the flow has to be:

1. Merge changes to the console chart
2. Release the console chart
3. Make updates to the redpanda chart and bump the version of console that it uses.
4. Release the redpanda chart

Reply:
I removed the changes from redpanda chart.